src/frontend/src/heartbeat.rs

```diff
@@ -49,8 +49,8 @@ impl HeartbeatTask {
     ) -> Self {
         HeartbeatTask {
             meta_client,
-            report_interval: heartbeat_opts.interval_millis,
-            retry_interval: heartbeat_opts.retry_interval_millis,
+            report_interval: heartbeat_opts.interval.as_secs(),
+            retry_interval: heartbeat_opts.retry_interval.as_secs(),
```

Comment:
```suggestion
            retry_interval: heartbeat_opts.retry_interval.as_millis(),
```

Reply:
thanks @killme2008 , updated in the latest commit